src/modules/search/components/Search.jsx

```diff
@@ -7,26 +7,45 @@ import SearchResults from './SearchResults';
 
 class Search extends React.Component {
 	handleClickOutside() {
+		this.props.toggleSearch();
```

Comment:
This has one unintended side effect. When you click back on the `Hide Search` button it is reopening the search. 

Can we stopPropagation() on the toggleSearch click?